.github/workflows/wheels.yml

```diff
@@ -0,0 +1,38 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build_wheels:
+    name: Build wheels for python ${{ matrix.python }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, windows-2019, macOS-11]
+        python: ['3.9', '3.10', '3.11', '3.12']
+
+    steps:
+      - uses: actions/checkout@v4
+
+      # Used to host cibuildwheel
+      - uses: actions/setup-python@v3
```

Comment:
I wonder if you even need the matrix of python versions here? From the minimal examples on the cibuildwheel site it looks like they just use the GHA they have created:

Is there any difference between installling via pip versus using the github action they have created? I see in their minimal examples that this is all just managed through the action

https://github.com/pypa/cibuildwheel/blob/796361d9f98c2b986f9d61bca9c116695aa5a1ca/examples/github-minimal.yml#L17

Maybe that with the python versions is all you need?

Reply:
Yeah, using the matrix explicitly in addition to the CIBW_BUILD apparently causes a duplication. It was an experiment to see if that would override and get around the 3.12 build issue. 

The sklearn, mypy, prophet, matplotlib, duckdb, and others listed all do things differently. Still searching for the MVP.

Reply:
Gotcha. Yea I'd just try the action instead of a matrix if we can. pantab is much simpler to build than the others - they might have more complicated reasons for needing the matrix

...assuming you haven't already